string class_label
int32 class_id
float32 xmin
float32 xmax
float32 ymin
float32 ymax
float32 score